apps/agreement/contracts/Agreement.sol

```diff
@@ -342,9 +342,9 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
         // Pay action submission fees
         Setting storage setting = _getSetting(currentSettingId);
         DisputableAragonApp disputable = DisputableAragonApp(msg.sender);
+        uint256 id = nextActionId++;
```

Comment:
Let's move this to be above the comment about "Pay action fees".

It could be a bit easy to miss this state change since it's not really relevant for paying app fees!